README.md

```diff
@@ -48,15 +48,15 @@
 
```

Comment:
![image](https://github.com/jcmunav63/school_library_oop/assets/93039359/9a34daea-9325-4d10-bdc0-53f9763776b1)

- [ ] It would be nice if you fix the highlighted links in the readme file which you can see in the image above 👀 because these links are broken 😨 Kindly fix them or remove them **(if you don't have any dedicated section for these broken links)** to make your readme descriptive and professional 👍